framework/src/executioners/FixedPointSolve.C

```diff
@@ -71,12 +64,40 @@ FixedPointSolve::validParams()
                                     "during fixed point iterations. This check is "
                                     "performed based on postprocessor defined by "
                                     "the custom_pp residual.");
-  params.addParam<bool>("direct_pp_value",
+
+  params.addParamNamesToGroup(
+      "accept_on_max_fixed_point_iteration fixed_point_rel_tol fixed_point_abs_tol "
+      "custom_pp direct_pp_value custom_abs_tol custom_rel_tol",
+      "Fixed point iterations");
+
+  return params;
+}
+
+InputParameters
+FixedPointSolve::validParams()
+{
+  InputParameters params = emptyInputParameters();
+  params += FixedPointSolve::fixedPointDefaultConvergenceParams();
+
+  params.addParam<unsigned int>(
+      "fixed_point_min_its", 1, "Specifies the minimum number of fixed point iterations.");
+  params.addParam<unsigned int>(
+      "fixed_point_max_its", 1, "Specifies the maximum number of fixed point iterations.");
+  params.addParam<bool>("disable_fixed_point_residual_norm_check",
                         false,
-                        "True to use direct postprocessor value "
-                        "(scaled by value on first iteration). "
-                        "False (default) to use difference in postprocessor "
-                        "value between fixed point iterations.");
+                        "Disable the residual norm evaluation thus the three parameters "
+                        "fixed_point_rel_tol, fixed_point_abs_tol and fixed_point_force_norms.");
+  params.addParam<bool>(
+      "fixed_point_force_norms",
+      false,
+      "Force the evaluation of both the TIMESTEP_BEGIN and TIMESTEP_END norms regardless of the "
+      "existence of active MultiApps with those execute_on flags, default: false.");
```

Comment:
why are these not moving to the fixed point convergence?

Reply:
They'd require an additional method/call on the Convergence object (e.g., `Convergence::preExecute()`), since the residual computations sandwich the solve in each iteration. However, I think it's the right design to move them - I'm gonna try it.